starlette/templating.py

```diff
@@ -12,7 +12,7 @@
     if hasattr(jinja2, "pass_context"):
         pass_context = jinja2.pass_context
     else:  # pragma: nocover
-        pass_context = jinja2.contextfunction
+        pass_context = jinja2.contextfunction  # type: ignore[attr-defined]
```

Comment:
Could you clarify why this is necessary? Is Jinja2 untyped / the types are wrong?

Reply:
The attribute is not defined on the latest `jinja2` (which is the one installed for Python 3.7+).

The conditional there is because Python 3.6 will have a previous `jinja2` version installed.

Reply:
Oh I see. It's just missing in newer versions of Jinja. And so MyPy complains that it's missing. Seems like a reasonable ignore.

Reply:
Can you use sys.version_info instead?

Reply:
I guess we could... I've added a note on the PR that removes Python 3.6 (https://github.com/encode/starlette/pull/1357) to remove this logic anyway.

Reply:
I thought about that but it seems like it could lead to issues because the coupling between the Python version and Jinja2 version is coming from our requirements file. If there were a `sys.version_info` but for jinja that _is_ what we would want to do